pom.xml

```diff
@@ -196,6 +196,18 @@
 			<version>1.2</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.slf4j</groupId>
```

Comment:
Will suppress the slf4j warning about not being able to find a provider (kind of required by Maven Testing Harness)